tensorflow/python/ops/math_ops.py

```diff
@@ -59,6 +59,7 @@
 @@log1p
 @@ceil
 @@floor
+@@rint
```

Comment:
We're trying to have numpy behavior @aselle 

Could you add this example to the unit test?
https://docs.scipy.org/doc/numpy/reference/generated/numpy.rint.html